app/views/candidate_interface/application_form/show.html.erb

```diff
@@ -1,12 +1,23 @@
-<% content_for :title, t('page_titles.application_form') %>
+<% content_for :title, @application_form.amendable? ? t('page_titles.edit_application_form') : t('page_titles.application_form') %>
 
 <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
-  <%= t('page_titles.application_form') %>
+  <%= @application_form.amendable? ? t('page_titles.edit_application_form') : t('page_titles.application_form') %>
 </h1>
-<p class="govuk-hint govuk-!-margin-bottom-8 govuk-!-margin-top-2"><%= @application_form_presenter.updated_at %></p>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <% if @application_form.amendable? %>
+      <div class="govuk-warning-text govuk-!-margin-top-6">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive">Warning</span>
+            <%= t('application_complete.edit_page.warning_text', remaining_days: formatted_days_remaining, date: edit_by_date) %>
```

Comment:
@paulrobertlloyd Did we want this to behave a bit more smartly when there's 24 hours or less left?

Reply:
i.e. to say hours remaining instead of 1 day (and then minutes remaining instead of 1 hour, etc.)

I can’t see this, so guess that’s what’s currently shown.

Reply:
When we've decided on an updated design, can it be added to the Trello card please (https://trello.com/c/IP7g0jBr/583-amending-an-application-after-submission and maybe as a comment)?

We can update it in a follow-up PR as we want to get this one merged in.